cmd/internal/codegen/parse/crd.go

```diff
@@ -215,28 +215,36 @@ func (b *APIs) parsePrimitiveValidation(t *types.Type, found sets.String, commen
 	return props, buff.String()
 }
 
+type mapTempateArgs struct {
+	Result string
+	SkipMap bool
+}
+
 var mapTemplate = template.Must(template.New("map-template").Parse(
 	`v1beta1.JSONSchemaProps{
     Type:                 "object",
-    AdditionalProperties: &v1beta1.JSONSchemaPropsOrBool{
+    {{if not .SkipMap}}AdditionalProperties: &v1beta1.JSONSchemaPropsOrBool{
```

Comment:
lets change it to SkipMapValidation to make it consistent.

Reply:
Done